src/components/data-management/ProjectDetails.jsx

```diff
@@ -341,6 +326,46 @@ const ProjectDetails = ({ width, height }) => {
     dispatch(deleteMetadataTrack(name, activeProjectUuid));
   };
 
+  const updateSpeciesOrMetadata = (value, metadataKey) => {
+    const updateObject = metadataKey === 'species' ? { species: value } : { metadata: { [metadataKey]: value } };
```

Comment:
what if `value` is `null`? Can `value` be `null`?

Reply:
no, the value can't be `null`. It'll be the value of the item that's selected in the dropdown.

Reply:
Inputting values into the dropdown to search does not update the `species` value in redux, so it doesn't corrupt the store.